test/test_support.ts

```diff
@@ -76,6 +77,11 @@ export function translateSources(contents: Input, options?: main.TranspilerOptio
   options = options || {};
   // Default to quick stack traces.
   if (!options.hasOwnProperty('failFast')) options.failFast = true;
+  const currentRunCompilerOpts = Object.assign({}, compilerOptions);
+  if (options.generateHTML) {
```

Comment:
nit: this logic shouldn't be duplicated in the test code and the app code. pass in a flag that has this effect instead of customizing it twice.